pages/configuration/server.conf.rst

```diff
@@ -0,0 +1,407 @@
+* ``web_enable_gzip = true``
+    * Enable/disable GZIP support for the web interface. This compresses HTTP responses and therefore helps to reduce overall round trip times. This is enabled by default.
+* ``web_enable_tls = false``
+    * Enable HTTPS support for the web interface. This secures the communication of the web browser with the web interface using TLS to prevent request forgery and eavesdropping.
+    * This is disabled by default. Set it to ``true`` to enable it and see the other related configuration settings.
+* ``web_tls_cert_file = /path/to/graylog-web.crt``
+    * The X.509 certificate chain file in PEM format to use for securing the web interface.
+* ``web_tls_key_file = /path/to/graylog-web.key``
+    * The PKCS#8 private key file in PEM format to use for securing the web interface.
+* ``web_tls_key_password = secret``
+    * The password to unlock the private key used for securing the web interface.
+* ``web_max_header_size = 8192``
+    * The maximum size of the HTTP request headers in bytes.
+* ``web_max_initial_line_length = 4096``
+    * The maximal length of the initial HTTP/1.1 line in bytes.
+* ``web_thread_pool_size = 16``
+    * The size of the thread pool used exclusively for serving the web interface.
+
+Elasticsearch
+^^^^^^^^^^^^^
+* ``elasticsearch_hosts = http://node1:9200,http://user:password@node2:19200``
+    * List of Elasticsearch hosts Graylog should connect to.
+    * Need to be specified as a comma-separated list of valid URIs for the http ports of your elasticsearch nodes.
+    * If one or more of your elasticsearch hosts require authentication, include the credentials in each node URI that requires authentication.
+    * Default: ``http://127.0.0.1:9200``
+* ``elasticsearch_connect_timeout = 10s``
+    * Maximum amount of time to wait for successfull connection to Elasticsearch HTTP port.
+    * Default: 10 seconds
+* ``elasticsearch_socket_timeout = 60s``
+    * Maximum amount of time to wait for reading back a response from an Elasticsearch server.
+    * Default: 60 seconds
+* ``elasticsearch_idle_timeout = -1s``
+    * Maximum idle time for an Elasticsearch connection. If this is exceeded, this connection will be tore down.
+    * Default: infinity
+* ``elasticsearch_max_total_connections = 20``
+    * Maximum number of total connections to Elasticsearch.
+    * Default: 20
+* ``elasticsearch_max_total_connections_per_route = 2``
+    * Maximum number of total connections per Elasticsearch route (normally this means per elasticsearch server).
+    * Default: 2
+* ``elasticsearch_max_retries = 2``
+    * Maximum number of times Graylog will retry failed requests to Elasticsearch.
+    * Default: 2
+* ``elasticsearch_discovery_enabled = false``
+    * Enable automatic Elasticsearch node discovery through Nodes Info, see `Elasticsearch Reference » Cluster APIs » Nodes Info <https://www.elastic.co/guide/en/elasticsearch/reference/5.4/cluster-nodes-info.html>`_.
```

Comment:
You might want to add that this setting must be false on AWS Elasticsearch Clusters (the hosted ones) and should be used carefully. 
In case of trouble with connections to ES this should be the first option to be disabled.

Reply:
Maybe you could also add a link to the Elasticsearch chapter of the documentation.